internal/sandbox/fspath/fspath.go

```diff
@@ -148,3 +148,7 @@ func TrimTrailingSlash(s string) string {
 func IsAbs(path string) bool {
 	return len(path) > 0 && path[0] == '/'
 }
+
+func IsLeaf(path string) bool {
+	return IndexSlash(path) < 0
+}
```

Comment:
I was curious about the usage of this, because I would consider a leaf on a file system to be something that has no children (i.e. a file rather than a directory), however in both cases there still might be a slash present in the path. I couldn't find any usage of the function to see what it was doing, so it could be removed?

Reply:
Ah yep, I used to need it but got rid of the code 👍